htdocs/comm/action/class/api_agendaevents.class.php

```diff
@@ -250,8 +250,19 @@ public function post($request_data = null)
 				continue;
 			}
 
+			// Special handling for fk_task to ensure it's properly saved
```

Comment:
This part of code seems to do the same things than the line 259
`$this->actioncomm->$field = $this->_checkValForAPI($field, $value, $this->actioncomm);`

do you really need it. Can you test without ?